pages/api/search/issues/index.ts

```diff
@@ -52,8 +54,22 @@ async function get(req: NextApiRequest, res: NextApiResponse) {
       });
 
       if (!network) return res.status(404).json("Invalid network");
-
+      networks = [network]
       whereCondition.network_id = network?.id;
+    } else {
```

Comment:
use `if (allNetworks)` instead, as we might fire this else condition in an edge case, and using a strict flag will solve that.